packages/icons-angular/README.md

```diff
@@ -0,0 +1,51 @@
+# `@carbon/icons-angular`
+
+> Angular components for icons in digital and software products using the Carbon
+> Design System.
+
+## Getting Started
+
+Run the following command using [npm](https://www.npmjs.com/):
+
+```bash
+npm install -S @carbon/icons-angular
+```
+
+If you prefer [Yarn](https://yarnpkg.com/en/), use the following command instead:
+
+```bash
+yarn add @carbon/icons-angular
+```
+
+## Usage
+
+Icons in this package support the following sizes: `16`, `20`, `24`, and `32`
+pixels. These sizes refer to the width and height of the icon. You can use an
+icon component into your project by doing the following:
+
+In your module:
+```ts
+import { IconModule } from "@carbon/icons-angular";
```

Comment:
Just a clarifying question, is this usage pattern able to be tree-shaken? Just wanted to double-check that including this wouldn't include all the unused icons in the final bundle 👍

Reply:
I was thinking about that and I think the answer is _no_ ... Unfortunately from what I can see the angular compiler doesn't tree shake ngmodule imports ... So I'll have to go to my backup plan of declaring a "ride along" NgModule for each icon, for maximum tree-shakability